pyperf/_compare.py

```diff
@@ -39,9 +39,9 @@ def format_normalized_mean(norm_mean):
     if norm_mean == 1.0:
         return "no change"
     elif norm_mean < 1.0:
-        return "%.2fx faster" % (1.0 / norm_mean)
```

Comment:
Maybe leave this as it is, but take a precision argument (which can default to 2).

Then in `format_geometric_mean()` call `format_normalized_mean(geo_mean, 3)`